stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/repository/ChannelRepository.kt

```diff
@@ -70,4 +70,14 @@ internal class ChannelRepository(
         channelCache.remove(cid)
         channelDao.setDeletedAt(cid, deletedAt)
     }
+
+    internal suspend fun setHiddenForChannel(cid: String, hidden: Boolean, hideMessagesBefore: Date?) {
```

Comment:
Does `hideMessagesBefore` need to be nullable? If yes maybe we can join those two methods into one with default parameter?

Reply:
Considering the data structure of the model it could be nullable, but in terms of an update, it doesn't make sense. You're right, I make argument non-nullable